ui/tract-renderer/src/main/java/org/cru/godtools/tract/analytics/model/ToggleLanguageAnalyticsActionEvent.kt

```diff
@@ -1,25 +1,22 @@
 package org.cru.godtools.tract.analytics.model
 
+import android.os.Bundle
 import java.util.Locale
 import org.ccci.gto.android.common.compat.util.LocaleCompat.toLanguageTag
-import org.cru.godtools.analytics.adobe.ADOBE_ATTR_LANGUAGE_SECONDARY
+import org.cru.godtools.analytics.firebase.PARAM_LANGUAGE_SECONDARY
 import org.cru.godtools.analytics.model.AnalyticsActionEvent
 import org.cru.godtools.analytics.model.AnalyticsSystem
 
-private const val ACTION_TOGGLE_LANGUAGE = "Parallel Language Toggled"
-
-private const val ADOBE_ATTR_TOGGLE_LANGUAGE = "cru.parallellanguagetoggle"
+private const val ACTION_TOGGLE_LANGUAGE = "parallel_language_toggled"
 
 class ToggleLanguageAnalyticsActionEvent(tool: String?, locale: Locale) :
     AnalyticsActionEvent(action = ACTION_TOGGLE_LANGUAGE) {
     override fun isForSystem(system: AnalyticsSystem) =
-        system === AnalyticsSystem.ADOBE || system === AnalyticsSystem.FACEBOOK
+        system === AnalyticsSystem.FIREBASE || system === AnalyticsSystem.FACEBOOK
 
     override val appSection = tool
     @OptIn(ExperimentalStdlibApi::class)
-    override val adobeAttributes = buildMap<String, Any> {
-        put(ADOBE_ATTR_TOGGLE_LANGUAGE, 1)
-        put(ADOBE_ATTR_LANGUAGE_SECONDARY, toLanguageTag(locale))
+    override val firebaseParams = Bundle().apply {
```

Comment:
```suggestion
    override val firebaseParams get() = Bundle().apply {
```

Reply:
After adding this I also had to wrap ```toLanguageTag(locale)``` with this instead ``` locale?.let { toLanguageTag(it) }``` because there was a type mismatch. Is there a better way to do this or is this alright?

Reply:
hmm, that's changing the locale that gets sent. we will need to preserve the `locale` constructor param, but don't want it to override `locale` from the base analytics event

Reply:
what it'll probably look like:

```kotlin
class ToggleLanguageAnalyticsActionEvent(tool: String?, private val selectedLocale: Locale) :
    AnalyticsActionEvent(action = ACTION_TOGGLE_LANGUAGE) {

    override val firebaseParams get() = Bundle().apply {
        putString(PARAM_LANGUAGE_SECONDARY, LocaleCompat.toLanguageTag(selectedLocale))
    }
```

Reply:
Ah, gotcha! Yeah, I tried just adding ```private val locale: Locale``` initially,  but it yelled at me. It makes sense why though.